x-pack/protocol/src/main/java/org/elasticsearch/protocol/xpack/graph/GraphExploreRequest.java

```diff
@@ -330,6 +359,41 @@ void writeTo(StreamOutput out) throws IOException {
         }
 
     }
-    
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        builder.startObject();
+        
+        builder.startObject("controls"); //=== Start Controls
```

Comment:
I've borrowed a style from dakrone that looks like
```
builder.startObject("controls");
{
  if (sampleSize != ... ) {
    builder.field("sample_size", sampleSize);
  }
  ...
}
builder.endObject();
```

It sort of replaces the need for `//=== Start Controls`.